### ISSUE TITLE
docbuild fail

### ISSUE THREAD

**Issue by martindurant** [fsspec/filesystem_spec#1209]
Two warnings:
```
/home/docs/checkouts/readthedocs.org/user_builds/filesystem-spec/checkouts/1188/docs/source/api.rst:109: WARNING: autosummary: failed to import fsspec.implementations.reference.ReferenceFileSystem.
Possible hints:
* AttributeError: module 'fsspec.implementations' has no attribute 'reference'
* ModuleNotFoundError: No module named 'numpy'
* AttributeError: module 'fsspec' has no attribute 'fsspec'
* ModuleNotFoundError: No module named 'fsspec.fsspec'
WARNING: autodoc: failed to import class 'reference.ReferenceFileSystem' from module 'fsspec.implementations'; the following exception was raised:
No module named 'numpy'
```

**Comment by ianthomas23** [comment]
I've had a look at this and confirmed that we can get rid of the `numpy` import error by adding `pandas` to the dependencies in `docs/environment.yml` (we need both `numpy` and `pandas` but `pandas` brings in `numpy` automatically).

But I don't understand the other errors here.

**Comment by ianthomas23** [comment]
That has solved the problem, so I'll close this issue.

**Comment by ianthomas23** [comment]
Fixed by #1224.
